src/DIRAC/Resources/IdProvider/OAuth2IdProvider.py

```diff
@@ -189,12 +202,28 @@ def get_metadata(self, option=None):
 
     @gMetadata
     def fetch_metadata(self):
-        """Fetch metada"""
+        """Fetch metadata"""
         if self.metadata_fetch_last < (time.time() - self.METADATA_REFRESH_RATE):
             data = self.get(self.server_metadata_url, withhold_token=True).json()
             self.metadata.update(data)
             self.metadata_fetch_last = time.time()
 
+    def getGroupScopes(self, group: str) -> list:
```

Comment:
I guess this is supposed to be:
```suggestion
    def getGroupScopes(self, group: str) -> list[str]:
```

Reply:
Corrected